### PR TITLE
feat(pf4-brand): introduce brand component

### PR DESCRIPTION
**Pull request by amarie401**
affects: @patternfly/react-core, @patternfly/react-docs

ISSUES CLOSED: #554  

#### What:
Introduced the brand component

#### Note:
Updated the versioning of react tokens pf4-core to be consistent with current version

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Brand/Brand.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  alt: PropTypes.string.isRequired
+};
+
+const defaultProps = {
+  className: '',
+  src: ''
+};
+
+const Brand = ({ className, src, alt, ...props }) => (
+  <img {...props} className={css('pf-c-brand', className)} src={src} alt={alt} />
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/Brand/Brand.js]: can you add a comment to say why we are using the 'pf-c-brand' string here, incase someone else is questioning it.

File: packages/patternfly-4/react-core/src/components/Avatar/Avatar.js [hunk 2]
```diff
@@ -4,15 +4,16 @@ import { css } from '@patternfly/react-styles';
 import PropTypes from 'prop-types';
 
 const propTypes = {
+  /** Additional classes added to the Avatar. */
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/Avatar/Avatar.js]: Change Avatar to Brand. Same comment for the other props.

**Comment by tlabaj** [comment]
@jgiardino What are wee doing for for accessibility? Should we be checking for the 'alt' attribute? Maybe generate warning if it missing? It probably depends on the purpose.  Decorative vs informational.

File: packages/patternfly-4/react-core/src/components/Avatar/Avatar.js (hunk 2 above)
Comment (amarie401) [reply packages/patternfly-4/react-core/src/components/Avatar/Avatar.js]: @tlabaj this is for the avatar component. I had the prop comments in the wrong spot in my last PR so I corrected it in this one.

Comment (tlabaj) [reply packages/patternfly-4/react-core/src/components/Avatar/Avatar.js]: Got it!

**Comment by amarie401** [comment]
@tlabaj the alt attribute is a required prop so I believe it will throw an error automatically if not given, same for the avatar component. We can generate a warning like we do with aria-label but may not be necessary. Thoughts @jgiardino ?

**Comment by tlabaj** [comment]
@amarie401 @jgiardino ignore my comment I see that the alt is a required attribute in the design.

**Pull request merged**